### PR TITLE
Submitting project for review

### PR DESCRIPTION
**Pull request by matheus-fls**

### CODE REVIEW

**Comment by bolah2009** [comment]
Currently under review

File: index.html [hunk 1]
```diff
@@ -2,26 +2,217 @@
...
+            <footer>
+                <div class="footer-links">
+                    <ul class="flex-row-cont links">
+                        <li><a href="#">F</a></li>
+                        <li><a href="#">T</a></li>
+                        <li><a href="#">Y</a></li>
+                        <li><a href="#">I</a></li>
+                        <li><a href="#">P</a></li>
+                        <li><a href="#">F</a></li>
+                        <li><a href="#">E</a></li>
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

Consider using social media icons.

File: index.html [hunk 2]
```diff
@@ -2,26 +2,217 @@
...
+                        <li><a href="#">JOBS</a></li>
+                        <li><a href="#">CONTACT</a></li>
+                    </ul>
 
-    <footer>
+                </div>
 
-    </footer>
+                <div class="footer-credits">
+                    <p>Kudos to the awesome devs at <a href="https://www.stackpath.com/">Stackpath</a> who made the original page.</p>
```
Comment (bolah2009) [inline index.html]: `SUGGESTION`

Use the content on the footer instead. See below:

tnw footer

**Review by bolah2009 — CHANGES_REQUESTED** [review]
### Hi Team :wave: 

Nice job so far, the [clone](https://raw.githack.com/matheus-fls/TNW-Clone/development/index.html) looks nice and responsive. :clap: :+1: 

However. there are some sections in the [original site](https://thenextweb.com/) that are missing, kindly add those, make them responsive and submit for another review.

##### Please note that this review may not be comprehensive enough because the project is not yet complete.

Good luck with your next submission.

Cheers
[Bola](https://bolabuari.com/)

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Way to go team 👏
Nice job building the responsive page. I'm giving this project a 👍. Go on and merge to master. Good luck with the next one.

**Pull request merged**